src/components/sessions/attendance-table/AttendanceTable.tsx

```diff
@@ -241,6 +254,14 @@ const AttendanceTable = ({ classObj, isEditing, onSubmit }: Props) => {
     );
     return [
       {
+        // FAMILY_ID_DATA_INDEX = 0
```

Comment:
can we remove this :0

Reply:
ooh i set these so that we don't miss updating the values if we change the order for some reason - updated the comment so this is clearer!